packages/alsa_lib.rb

```diff
@@ -3,22 +3,14 @@
 class Alsa_lib < Package
   description 'The Advanced Linux Sound Architecture (ALSA) provides audio and MIDI functionality to the Linux operating system.'
   homepage 'https://www.alsa-project.org/main/index.php/Main_Page'
-  version '1.2.2'
+  version '1.2.3.2'
```

Comment:
version '1.2.3'?

Reply:
Nope : latest release is really 1.2.3.2... I just made a "too-quick $#@!§ " copy/paste... Sorry for that. See correction of url / sha

Reply:
Gotcha. LoL